pkgs/development/python-modules/simpy/default.nix

```
@@ -1,4 +1,5 @@
-{ buildPythonPackage, fetchPypi, lib, setuptools_scm, pytestCheckHook }:
+{ buildPythonPackage, fetchPypi, lib, setuptools, setuptools_scm
+, pytestCheckHook }:
 
 buildPythonPackage rec {
   pname = "simpy";
```

should be added after version, just can't see in github diff
```suggestion
  pname = "simpy";
  disabled = isPy27;
```

```
builder for '/nix/store/vhac4469zyak4w2ab7s2w1vyql8k1rpc-python2.7-simpy-4.0.1.drv' failed with exit code 1; last 10 log lines:
  adding 'simpy-4.0.1.dist-info/top_level.txt'
  adding 'simpy-4.0.1.dist-info/RECORD'
  removing build/bdist.linux-x86_64/wheel
  Finished executing setuptoolsBuildPhase
  installing
  Executing pipInstallPhase
  /build/simpy-4.0.1/dist /build/simpy-4.0.1
  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Processing ./simpy-4.0.1-py2.py3-none-any.whl
  ERROR: Package 'simpy' requires a different Python: 2.7.18 not in '>=3.6'
```

---

I've disabled Python 2 compatibility, since Python 3.6 is the minimum supported version.